.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I don't think we should add prettier at the same time as a node upgrade, maybe a separate PR

Reply:
I agree. @george-the-wizard please focus on upgrading the code to be compatible with Node 16 and any additional work can be followed up later.

Reply:
@miniscruff @rahulreddy I needed it in order to be able to troubleshoot the issues efficiently, I will revert the related changes after it builds flawlessly - thx